src/ldp/auxiliary/AuxiliaryStrategy.ts

```diff
@@ -10,7 +10,7 @@ import type { AuxiliaryIdentifierStrategy } from './AuxiliaryIdentifierStrategy'
  */
 export interface AuxiliaryStrategy extends AuxiliaryIdentifierStrategy {
   /**
-   * Whether this auxiliary resource in a root storage container.
+   * Whether this auxiliary resource is required in a root storage container.
```

Comment:
I'm not sure if I understood this sentence correctly, but do we mean this?

```suggestion
   * Whether the root storage container requires this auxiliary resource to be present.
```

Reply:
Pretty much. The current auxiliary implementation is based on what is needed for acl files and might be overtuned a bit for that, but one of the features there is that you can't delete an acl file that is stored in a `pim:Storage` container.